### PR TITLE
Advanced Transformations beta updates

### PR DESCRIPTION
**Pull request by rchinn-segment**
### Proposed changes
- Added Advanced Transformations beta updates
- Linked API transformations to PAPI docs
- Other miscellaneous edits

### Merge timing

- Deploy 11/30 (along with the [PAPI doc intro](https://github.com/segmentio/public-api/pull/1022))

### CODE REVIEW

**Review by ranand — APPROVED** [review]

File: src/protocols/transform/index.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ redirect_from: '/protocols/transformations/'
 
 ## What is a Transformation?
 
-Transformations allow you to change data as it flows through Segment to either correct bad data or customize data for a specific destination. At this time, you can change event and property names to align events with your Tracking Plan, or to conform to a destination-specific requirement. For example, a Transformation could be created to change an event name from `completed_order` to `Order Completed` to conform to [Segment's ecommerce spec](/docs/connections/spec/ecommerce/v2/#order-completed).
+With Transformations, you can change data as it flows through Segment to either correct bad data or customize data for a specific destination. Change event and property names to align events with your Tracking Plan, or to conform to a destination-specific requirement. For example, a Transformation could be created to change an event name from `completed_order` to `Order Completed` to conform to [Segment's ecommerce spec](/docs/connections/spec/ecommerce/v2/#order-completed).
```
Comment (markzegarelli) [inline src/protocols/transform/index.md:10]:
```suggestion
With Transformations, you can change data as it flows through Segment to either correct bad data or customize data for a specific destination. Change event and property names to align events with your Tracking Plan, or to conform to a destination-specific requirement. For example, you can create a Transformation to change an event name from `completed_order` to `Order Completed` to conform to [Segment's ecommerce spec](/docs/connections/spec/ecommerce/v2/#order-completed).
```

Let's make this active voice.

File: src/protocols/transform/index.md [hunk 2]
```diff
@@ -21,15 +24,15 @@ Segment's goal is to make Transformations a powerful tool that complements a wel
 ### Other important notes
 
 - **Transformations cannot be applied retroactively:** They only apply to data moving forward. However, you can manually extract and re-send (or even [Replay](/docs/guides/what-is-replay)) events through a source with an active destination Transformation, which will send the transformed events to your destinations.
-- **Transformations are only available to Protocols customers:** If you are interested in this feature, contact your Account Executive or CSM to learn more about the Protocols package.
+- **Transformations are only available to Protocols customers:** If you're interested in this feature, contact your Account Executive or CSM to learn more about the Protocols package.
```
Comment (markzegarelli) [inline src/protocols/transform/index.md:27]:
```suggestion
- **Transformations are available to Protocols customers:** If you're interested in this feature, contact your Account Executive or CSM to learn more about the Protocols package.
```

File: src/protocols/transform/index.md [hunk 3]
```diff
@@ -43,11 +46,11 @@ To create a Transformation, navigate to the Transformations tab in Protocols and
 > Workspace Owner or Source Admin permissions are required to create and edit transformations.
 > Source Read-only permissions are required to view transformations.
 
-![](../images/transformation_wizard.png)
+![create a transformation with the three-step wizard](../images/transformation_wizard.png)
 
 ### Step 1: Select the transformation type
 
-To create a Transformation, you first need to select which type of transformation you want to create. For each transformation type, Segment displays a description, use cases and example payload. Current transformation types include:
+To create a Transformation, you first need to select which type of transformation you want to create. For each transformation type, Segment displays a description, use cases, and example payload. Current transformation types available in your Segment workspace include:
```
Comment (markzegarelli) [inline src/protocols/transform/index.md:53]: oxford comma, 👨‍🍳 💋

File: src/protocols/transform/index.md [hunk 4]
```diff
@@ -86,12 +92,34 @@ Depending on the type of transformation you selected, you will need to enter the
...
+
+- **Rename an event:** Change an event name from `viewed_product` to `Product Viewed`.
+
+- **Rename a property or trait:** Change the property name `revenue` to `total` for a specific destination.
+
+- **Update a property value:** Use [Segment's Public API](https://docs.segmentapis.com/tag/Transformations){:target="_blank"} to transform the property `currency` to have the value `USD`. This transformation is in beta.
+
+- **Add a new property name and assign a value:** If you want to create a new property and set a static value, use [Segment's Public API](https://docs.segmentapis.com/tag/Transformations){:target="_blank"} to create `new_property: static_value`. This transformation is in beta.
+
+<!-- hiding this transformation until it's available
```
Comment (markzegarelli) [inline src/protocols/transform/index.md:117]: Can you use liquid comments here? HTML comments still renders the content, but just prevents it from displaying on the page. Liquid comments block the commented section from being rendered to HTML.

```
{% comment %}
content
{% endcomment %}
```

File: src/protocols/transform/index.md [hunk 5]
```diff
@@ -86,12 +92,34 @@ Depending on the type of transformation you selected, you will need to enter the
...
+- **Change property value casing:** Transform property value casing to lowercase, uppercase, or title case. For example, Transform the property value `united states` to `USA` to remain consistent with your data tracking.
+--> 
+
+> note ""
```
Comment (markzegarelli) [inline src/protocols/transform/index.md:121]:
```suggestion
> info ""
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**